corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -114,9 +114,11 @@
 <div class="row-fluid">
     <div class="span12">
         <div class="btn-toolbar">
-            <a class="btn btn-info" href="{% url "location_import" domain %}">
-                <i class="icon-upload"></i> {% trans 'Bulk Import Locations' %}
-            </a>
+            {% if has_location_types %}
+                <a class="btn btn-info" href="{% url "location_import" domain %}">
+                    <i class="icon-upload"></i> {% trans 'Bulk Import Locations' %}
```

Comment:
not sure I agree with that. @amsagoff what's your reasoning? Here's mine: http://www.joelonsoftware.com/items/2008/07/01.html (the tooltip I think supercedes the argument against disabling and is better UX)